test/__tests__/bmap_test.ml

```diff
@@ -0,0 +1,102 @@
+open Test
+
+module C = Containers
+(* Int64 module tests *)
+
+(* Intentionally not tested as Int64 module is just a wrapper around stdlib, this should be part of BS testing *)
+
+
+(* Map module tests *)
+module IntMap = C.Map.Make(Int64)
+
+let to_int64 i = match C.Int64.of_int i  with
+  | Some x -> x
+  | None -> raise (Failure "Corrupted implementation of Int64.of_int")
+
+let map0 = IntMap.add (to_int64 1) "one" IntMap.empty
+
+let () = test "map.size" (fun t -> 
+    let x = IntMap.size IntMap.empty in
+    deepEqual t x 0
+  )
+
+let () = test "map.size" (fun t -> 
+    let x = IntMap.size map0 in
+    deepEqual t x 1
+  )
+
+let () = test "map.get" (fun t ->
+    let x = IntMap.get (to_int64 1) map0 in match x with
+    | Some str -> deepEqual t str "one"
+    | None -> fail t
```

Comment:
Wouldn't it be better to just test the value directly with `deepEqual t x (Some "one")` and drop the pattern match?